### PR TITLE
Spark 2.1

### PR DESCRIPTION
**Pull request by ookagan**
Hi Samuel, this is the current version of the project that compiles and runs against Spark 2.1 and Scala 2.11.8

Depends on upgraded version of the salesforce-wave-api, which I will also share with you.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,92 +1,57 @@
...
-scalaVersion := "2.10.4"
-
-resolvers += "sonatype-snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/"
-
-libraryDependencies ++= Seq(
-  "com.force.api" % "force-wsc" % "37.0.3",
-  "com.force.api" % "force-partner-api" % "37.0.3",
-  "com.springml" % "salesforce-wave-api" % "1.0.7",
-  "org.mockito" % "mockito-core" % "2.0.31-beta"
-)
```
Comment (springml) [inline build.sbt]: Could you please revert the changes done in build.sbt?

**Comment by ookagan** [comment]
Sam, I am assuming you can see and accept the changes in the build.sbt.
For some reason I could not run the build file with the settings you had for sbt-spark-package, so I commented those out.  I am assuming you can pull those updates with the pull request I had created earlier. Please lmk otherwise, so I can create another one.

**Comment by springml** [comment]
@ookagan Thanks for your pull request. Added support for Spark 2.0
